ebml/EbmlUnicodeString.h

```diff
@@ -55,17 +55,15 @@ class EBML_DLL_API UTFstring {
   using value_type = wchar_t;
 
   UTFstring() = default;
-  UTFstring(const wchar_t *); // should be NULL terminated
-  UTFstring(const UTFstring &);
-  UTFstring(std::wstring const &);
+  explicit UTFstring(const wchar_t *); // should be NULL terminated
+  explicit UTFstring(std::wstring const &);
 
   virtual ~UTFstring();
   bool operator==(const UTFstring&) const;
   inline bool operator!=(const UTFstring &cmp) const
   {
     return !(*this == cmp);
   }
-  UTFstring & operator=(const UTFstring &);
```

Comment:
Why exactly do you want to remove the copy c'tor & assignment operator for `UTFstring`? What's the benefit?

Reply:
I think what SonarLint wants here is rule of 0 semantics.